terra/tests/utils.py

```diff
@@ -87,17 +116,29 @@ def tearDown(self):
 
 
 class TestComputeCase(TestCase):
+  '''
+  Test case that mocks for ``_connection`` in ``terra.compute.utils.compute``.
+  This allows for a compute to be retrieved from the ``ComputeHandler`` for a
+  single test. More useful when used in :class:`TestLoggerConfigureCase`
+  '''
+
   def setUp(self):
-    import terra.compute.utils
-    self.patches.append(mock.patch.dict(terra.compute.utils.compute.__dict__))
+    from terra.compute.utils import compute
+    self.patches.append(mock.patch.dict(compute.__dict__))
     super().setUp()
 
 
 class TestExecutorCase(TestCase):
+  '''
+  Test case for that mocks for ``_connection`` in
+  ``terra.executor.utils.Executor``. This allows for an executor to be
+  retrieved from the ``ExecutorHandler`` for a single test. More useful when
+  used in conjunction with :class:`TestLoggerConfigureCase`
```

Comment:
```suggestion
  Test case allows for an executor to be retrieved from the ``ExecutorHandler``
  for a single test. This is done by mocking ``_connection`` in
  ``terra.executor.utils.Executor``, so that the Handler does not stay
  initialized. This  More useful when used in conjunction with
  :class:`TestLoggerConfigureCase`
```